examples/resources/awscc_gamelift_build/gamelift_build.tf

```diff
@@ -0,0 +1,61 @@
+// Create new Amazon GameLift Build
+resource "awscc_gamelift_build" "example" {
+  name             = "example-build"
+  version          = "1"              // (Optional) - used for versioning your GameLift Builds
```

Comment:
Just suggestion, probably remove all the comments in the .tf file as the Terraform doco contains fair bit of description for each schema. If there is specific thing you would like to call out, probably worth to mention in the gamelift_build.md.tmpl with specific example so that to keep the code clear.